wire-ios-request-strategy/Sources/Notifications/NotificationStreamSync.swift

```diff
@@ -40,12 +40,19 @@ public class NotificationStreamSync: NSObject, ZMRequestGenerator, ZMSimpleListR
         self.lastEventIDRepository = eventIDRespository
         super.init()
         managedObjectContext = moc
-        listPaginator = ZMSimpleListRequestPaginator.init(basePath: "/notifications",
-                                                          startKey: "since",
-                                                          pageSize: 500,
-                                                          managedObjectContext: moc,
-                                                          includeClientID: true,
-                                                          transcoder: self)
+
+        let selfUser = ZMUser.selfUser(in: moc)
+        let selfClientID = selfUser.selfClient()?.remoteIdentifier
+
```

Comment:
are we sure it's executed on the moc context here ?

Reply:
for me it's executed on the thread from :
```
   public override func didReceive(
        _ request: UNNotificationRequest,
        withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void
```

Reply:
Yep I think you're right, fixed 366b05e4445c45107cc31d79ca1defb49e86173f